Game/items.gd

```diff
@@ -0,0 +1,6 @@
+extends Node
+
+@export var items_dict: Dictionary[Item.Type, ItemResource]
+
+func get_info(type: Item.Type) -> ItemResource:
```

Comment:
"get_info" is not a very descriptive name. Can you add a documentation comment explaining the context of this function better?